Release notes for v1.3 (#5470)
gammapy/gammapy: docs/release-notes/v1.3.rst

.. include:: ../references.txt

.. _gammapy_1p3_release:

1.3 (October xx, 2024)
-------------------------

- Released October xx, 2024
- xx contributors
- 160 pull requests since v1.2 (not all listed below)
- 59 closed issues

Contributors
~~~~~~~~~~~~
- Stefan Fröse
- REGEARD Maxime
- Bruno Khélifi
- Claudio Galelli

> - aaguasca

Author: @aaguasca Can you please update your full name here, and for the future, ensure your git credentials are properly configured (https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address)?

Reviewer: Thank you very much for spotting this. My full name is Arnau Aguasca-Cabot. Could you please give me permission to AtreyeeS/gammapy.git to edit the file myself?

- Fabio PINTORE
- Daniel Morcuende
- Atreyee Sinha
- Brigitta Sipőcz
- Michele Peresano
- Maximilian Linhoff

> - AOI

Author: @GerritRo Can you please update your full name here, and for the future, ensure your git credentials are properly configured (https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address)?

Reviewer 2: ```suggestion
- Gerrit Roellinghoff
```

> - stahanna

Author: @stahanna Can you please update your full name here, and for the future, ensure your git credentials are properly configured (https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address)?

.. include:: ../references.txt

.. _gammapy_1p3_release:

1.3 (October xx, 2024)
-------------------------

- Released October xx, 2024
- xx contributors
- 160 pull requests since v1.2 (not all listed below)
- 59 closed issues

Contributors

> ~~~~~~~~~~~~

Reviewer 3: Should be changed to alphabetic order

- Stefan Fröse

> - REGEARD Maxime

Reviewer 4: ```suggestion
- Maxime Regeard
```

- Bruno Khélifi
- Claudio Galelli

> - aaguasca

Author: ```suggestion
- Arnau Aguasca-Cabot
```

> - aaguasca

Reviewer 5: ```suggestion
- Arnau Aguasca-Cabot
```

- Fabio PINTORE
- Daniel Morcuende
- Atreyee Sinha
- Brigitta Sipőcz
- Michele Peresano

> - Maximilian Linhoff

Reviewer 4: ```suggestion
- Maximilian Linhoff
- Hanna Stapel
- Pei Yu
```

- AOI

> - stahanna

Reviewer 5: ```suggestion
- Hanna Stapel
```

- Quentin Remy

> - Astro-Kirsty

Reviewer 4: ```suggestion
- Kirsty Feijen
```

.. include:: ../references.txt

.. _gammapy_1p3_release:

1.3 (October xx, 2024)
-------------------------

- Released October xx, 2024
- xx contributors
- 160 pull requests since v1.2 (not all listed below)
- 59 closed issues

Contributors
~~~~~~~~~~~~
- Stefan Fröse
- Maxime Regeard
- Bruno Khélifi
- Claudio Galelli
- Arnau Aguasca-Cabot

> - Fabio PINTORE

Reviewer 6: ```suggestion
- Fabio Pintore
```

- Daniel Morcuende
- Atreyee Sinha
- Brigitta Sipőcz
- Michele Peresano
- Maximilian Linhoff
- Hanna Stapel
- Pei Yu
- AOI
- stahanna
- Quentin Remy
- Kirsty Feijen
- Fabio Acero
- Régis Terrier
- Lars Mohrmann

> - VascoSch92

Reviewer 6: We also need this one!

Author: @VascoSch92 Can you please update your full name here, and for the future, ensure your git credentials are properly configured (https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address)?

> - VascoSch92

Reviewer 7: ```suggestion
- Vasco Schiavo
```

Reviewer 6: Hello @VascoSch92. Could you please provide a contact email so that we can email you in regards to the gammapy v2.0 paper?

- Laura Olivera Nieto
- Axel Donath
- Samantha Wong

Summary
~~~~~~~

This release introduces a number of performance improvements, new features and bug fixes.
Improved support is provided for joint analysis with different event types.
There has been significant code cleanup, specially with respect to the documentaion and three new tutorials have been added.

This release is compatible with numpy >=2.0. 
Sherpa dependency has beeen fixed to >4.16

API changes
~~~~~~~~~~~

> One deprecated change has been introducted in this version:

Reviewer 6: There may also be another deprecation for this one #5496

> - The order of arguments in the `FoVBackgroundModel` has been modified; `daatset_name` is now a complusoryt first positional argument.

Reviewer 6: ```suggestion
- The order of arguments in the `FoVBackgroundModel` has been modified; `dataset_name` is now a compulsory first positional argument.
```

.. include:: ../references.txt

.. _gammapy_1p3_release:

1.3 (October xx, 2024)
-------------------------

- Released October xx, 2024
- xx contributors
- 160 pull requests since v1.2 (not all listed below)
- 59 closed issues

Summary
~~~~~~~

This release introduces a number of performance improvements, new features and bug fixes.
There has been significant code cleanup, specially with respect to the documentaion and three new tutorials have been added.

> This release is based on numpy 1.2 and css for sphinx v15 (REPHRASE)

Reviewer 4: It is not numpy 2.1.1?

API changes
~~~~~~~~~~~

No new API breaking changes have been introducted in this version.

> Fetures deprecated since Gammapy v1.2 have been removed. 

Reviewer 4: ```suggestion
Features deprecated since Gammapy v1.2 have been removed. 
```

New features
~~~~~~~~~~~~

*gammapy.catalogs*

> - Support for the 2PC and 3PC pulsar catalogs from Fermi-LAT is now added

Reviewer 4: ```suggestion
- Support for the 2PC and 3PC pulsar catalogs from Fermi-LAT is now added.
```

*gammapy.datasets*

> - Performance improvement in Fermi-LAT analysis with adaption in psf kernel computation

Reviewer 4: ```suggestion
- Performance improvement in Fermi-LAT analysis with adaption in psf kernel computation.
```

> - Stacking of acceptance maps improved

Reviewer 4: ```suggestion
- Stacking of acceptance maps improved.
```

*gammapy.maps*

> - Multiple non-spatial axes supported in creation on MapDatasets. Full likelihood computation with additional axes not yet supported

Reviewer 4: ```suggestion
- Multiple non-spatial axes supported in creation on MapDatasets. Full likelihood computation with additional axes not yet supported.
```

- Map axes with periodic boundary conditions are now supported.
- An error is raised for maps with invalid input shapes instead of silently broadcasting.

*gammapy.estimators*

- Binwise likelihood profiles are now stored on FluxMaps. FluxMaps obtained from different datasets can be combined.
- A new tutorial explaining the general Estimator API has been added.

> - A new tutorial showing how to perform time dependent spectroscopy has been added

Reviewer 4: ```suggestion
- A new tutorial showing how to perform time dependent spectroscopy has been added.
```

> - A new tutorial showing how to perform energy dependent spectroscopy has been added

Reviewer 8: ```suggestion
- A new tutorial showing how to perform time dependent spectroscopy has been added
```

*gammapy.stats*

- Implementation of the Timmer-Koenig algorithm with leakage protection to simulate a time series from a power spectrum.
- Computation of the discrete structure function for a variable source using Emmanoulopoulos et al. (2010) algorithm.
- Computation of the discrete cross correlation function added.

> 

Reviewer 4: ```suggestion
*infrastructure*

- Add Matomo as website analytics, instead of Google Analytics.

```

*gammapy.modeling*

- Unique naming of model parameters adapted, leading to resolution of issues with computation of the covariance matrix for complex model lists

> - New tutorial added for modeling EBL absorption

Reviewer 4: ```suggestion
- New tutorial added for modeling EBL absorption.
```
